miniapps/nurbs/CMakeLists.txt

```diff
@@ -9,33 +9,59 @@
 # terms of the BSD-3 license. We welcome feedback and contributions, see file
 # CONTRIBUTING.md for details.
 
-configure_file(${PROJECT_SOURCE_DIR}/miniapps/nurbs/square-nurbs.mesh
-               ${PROJECT_BINARY_DIR}/miniapps/nurbs/square-nurbs.mesh
-               COPYONLY)
+#configure_file(${PROJECT_SOURCE_DIR}/miniapps/nurbs/square-nurbs.mesh
```

Comment:
Why are these commented out?

Reply:
The testcases below now use the full path to the data directory, so there is no need for a copying the files. I will remove the lines all together to prevent confusion.